### ISSUE TITLE
Excessive spawning

### ISSUE THREAD

**Issue by kevdevrev** [RayTrace082/untamedwilds#54]
I have hundreds of tortises and snakes spawning in the plains biome, and no matter what I do I cannot exterminate them.
I turned off snakes in my config, and disabled breeding (only by play) and they continue to spawn, killing colonists and villagers, and myself. I have killed at least 200 in the past hour. I don't know if the config options are working, and I read that they spawn per chunk but it really seems like they are just popping in like a zombie or skeleton would at night.

Basically, disable snakes is not working.

**Comment by RayTrace082** [comment]
Disabling stuff from the config only works when it comes to world-gen, they will not remove existing entities in the world
There was a (fixed in-dev) bug that caused burrows to accumulate massive amounts of creatures inside, so this may be the reason why this happened

You should be able to use the following command:
`/kill @e[type=untamedwilds:snake]`
To kill all snakes in the currently loaded chunks, do note that you should be also be breaking any burrow containing snakes (right click them with an empty hand to know what mob lives inside) because they will constantly spawn new snakes

**Comment by kevdevrev** [comment]
Finding a barrow in a grassy plains biome is like looking for a needle in a haystack. This isn't playable and I just realized the server issues I was having was probably due to the 300+ snakes always roaming at my base at any given time.
